spyder/plugins/breakpoints/__init__.py

```diff
@@ -3,4 +3,4 @@
 #==============================================================================
 # The following statement is required to register this 3rd party plugin:
 #==============================================================================
-from .breakpoints import Breakpoints as PLUGIN_CLASS
```

Comment:
Please remove this line from all plugins. This is not necessary anymore because they are now core plugins.

Reply:
I think instead of removing it from these plugins, we could add it to other internal plugins, also with the variables for config files that @goanpeca suggested in https://github.com/spyder-ide/spyder/issues/4591#issuecomment-330835315

I prefer that internal and external plugins have the same structure, so in a future we could refactor and load all plugins in the same way.

Reply:
I agree with @rlaverde

Reply:
We discussed this with @rlaverde yesterday and agreed that it'd be nice to have for all plugins indeed.